config/src/config/mempool_config.rs

```diff
@@ -46,6 +46,8 @@ pub struct MempoolConfig {
     pub shared_mempool_peer_update_interval_ms: u64,
     /// Interval to update peer priorities in shared mempool (seconds).
     pub shared_mempool_priority_update_interval_secs: u64,
+    /// The amount of time to wait after transaction insertion to broadcast to a failover peer.
```

Comment:
now that we have delay, should we broadcast to 2 failover peers? or broadcast to one , and after another delay to second one? 

that might cut down on p90 quite a bit

Reply:
There's some risk that it could increase duplicates in overload scenarios (when even the relaxed quorum store backpressure kicks in). I'll first land this and think about it some more...